apache-beam/ctr_prediction/.gitignore

```diff
@@ -0,0 +1,5 @@
+.DS_Store
+.idea
+*.iml
```

Comment:
These should already be covered from the main ignore

Reply:
removed as main ignore has them